requirements/dev.in

```diff
@@ -0,0 +1,15 @@
+-r prod.txt
+black
+coverage
+coveralls
+flake8
+myst-parser
+pip-tools
+pre-commit
+pytest
+pytest-cov
+Sphinx==3.5.4      # pin this until Myst is happy with Sphinx 4
```

Comment:
Use `Sphinx<4` here so that you still get minor updates within the 3.x series.

Reply:
I'm actually surprised you needed to do this since myst-parser's setup.cfg declares sphinx<4 and pip should take that into account when resolving.